tests/Backend/CommonPart1/ImportExportCommonTest.php

```diff
@@ -58,8 +58,21 @@ public function testTableImportExport(CsvFile $importFile, $expectationsFileName
      * @dataProvider tableImportData
      * @param $importFileName
      */
-    public function testTableAsyncImportExport(CsvFile $importFile, $expectationsFileName, $colNames, $format = 'rfc', $createTableOptions = array())
-    {
+    public function testTableAsyncImportExport(
+        CsvFile $importFile,
+        $expectationsFileName,
+        $colNames,
+        $format = 'rfc',
+        $createTableOptions = [],
+        $testCaseDescription = ''
+    ) {
+        if ($testCaseDescription === 'utf8BOM') {
+            $tokenData = $this->_client->verifyToken();
+            if ($tokenData['owner']['defaultBackend'] === self::BACKEND_EXASOL) {
+                self::markTestSkipped('Exasol doesnt support files with BOM');
+                return;
+            }
+        }
```

Comment:
Asi cajk možná by sa mě víc lůbilo kdy to bylo místo $testCaseDescription třeba $ignoredBackends

Reply:
zrusil jsem